Library/Formula/ghc.rb

```diff
@@ -1,8 +1,8 @@
 class Ghc < Formula
   desc "Glorious Glasgow Haskell Compilation System"
   homepage "https://haskell.org/ghc/"
-  url "https://downloads.haskell.org/~ghc/7.10.3/ghc-7.10.3a-src.tar.bz2"
-  sha256 "877899988b64d13a86148f4db29b26d5adddef9207f718b726dc5c202d8efd8e"
+  url "https://downloads.haskell.org/~ghc/7.10.3/ghc-7.10.3b-src.tar.bz2"
```

Comment:
It's what the main upstream download page is pointing at now. https://www.haskell.org/ghc/download_ghc_7_10_3#sources

Referenced on mailing list here: http://osdir.com/ml/glasgow-haskell-users@haskell.org/2015-12/msg00011.html. Looks like it's just another issue with the release notes included in the distribution.